// mui-label-divider/dist/index.js
function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var core = require('@material-ui/core');
var React = _interopDefault(require('react'));

var useStyles = core.makeStyles(function (theme) {
  return {
    label: {
      overflow: "hidden",
      textAlign: "center",
      "&::before, &::after": {
        backgroundColor: theme.palette.divider,
        content: '""',
        display: "inline-block",
        height: "1px",
        position: "relative",
        verticalAlign: "middle",
        width: "50%"
      },
      "&::before": {
        right: theme.spacing(1),
        marginLeft: "-50%"
      },
      "&::after": {
        left: theme.spacing(1),
        marginRight: "-50%"
      }
    }
  };
});
var LabelDivider = function LabelDivider(_ref) {
  var label = _ref.label;
  var classes = useStyles();
  return React.createElement(core.Typography, {
    variant: "subtitle2",
    color: "textSecondary",
    align: "center",
    className: classes.label
  }, label);
};

exports.LabelDivider = LabelDivider;
